02/app.js

```diff
@@ -1 +1,11 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const dataUrl = Array.from(document.querySelectorAll("[data-url]"));
+
+const dataUrlValue = dataUrl.map(function (element) {
```

Comment:
Tutaj nie musimy sprawdzać bo `querySelectorAll` zwraca tablicę - jak będzie pusta to po prostu nic się nie wykona i nie będzie błędu :)